classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :title, :author, :rentals
+
+  def initialize(label)
+    @label = label
```

Comment:
- [ ] As in this app we intend to both _read_ and _modify_ `@label` let's ensure to mark it with `attr_accessor`, like below:
```ruby
class Classroom
  attr_accessor :label 
 
  def initialize(label)
    @label = label
    .
  end
end
```

Let's endeavor to read further on the same [here](`https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_principles_examples.md#useful-shortcuts-attr_reader-attr_writer-and-attr_accessor`)